### PR TITLE
Update packages versions

### PR DESCRIPTION
**Pull request by arikaiy**
The latest version works correctly on Windows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
     "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "devDependencies": {
-    "axios": "^0.15.3",
+    "axios": "^0.16.1",
     "bootstrap-sass": "^3.3.7",
-    "cross-env": "^3.2.3",
-    "jquery": "^3.1.1",
+    "cross-env": "^5.0.0",
```
Comment (tillkruss) [inline package.json]: Laravel Mix uses `cross-env` at `^3.1.3". Can you confirm with the Mix team?

Comment (arikaiy) [inline package.json]: Yes, of course. I sent them a pull request.

**Comment by taylorotwell** [comment]
I don't understand this PR. Don't our *current* constraint already match all of these versions you are updating to. If I specify I want ^3.1.1... is that not 3.1.1 up to and not including 4.0?

File: package.json (hunk 1 above)
Comment (duo51) [inline package.json]: [](url)

Comment (duo51) [inline package.json]: ``

Comment (duo51) [inline package.json]: [](url)

**Comment by arikaiy** [comment]
https://github.com/JeffreyWay/laravel-mix/pull/810#issuecomment-302123927

**Comment by taylorotwell** [comment]
I'm closing this since nobody has responded to my comment about our existing version constraints.

**Pull request closed without merging**